pubsub/cloud-client/src/main/java/com/example/pubsub/SubscriberExample.java

```diff
@@ -66,11 +81,12 @@ public static void main(String... args) throws Exception {
       subscriber =
           Subscriber.newBuilder(subscriptionName, new MessageReceiverExample()).build();
       subscriber.startAsync().awaitRunning();
-      // Continue to listen to messages
+      // Continue to listen to queue
       while (true) {
-        MessageAndConsumer messageAndConsumer = messages.take();
+        MessageAndConsumer messageAndConsumer = queue.take();
         System.out.println("Message Id: " + messageAndConsumer.message.getMessageId());
         System.out.println("Data: " + messageAndConsumer.message.getData().toStringUtf8());
+        // Ack only once all work with the message is complete.
```

Comment:
nit: Should it be "Ack only once [after] all work.."?

Reply:
I think the "once" in there is confusing now that I read it, so I changed it.